quickstart/.travis.yml

```diff
@@ -9,5 +9,4 @@ tex-config:
 - packages=lipsum, blindtext
 
 script:
-- docker pull strauman/travis-latexbuild:small # select small texlive scheme
```

Comment:
This should still be here, or no?

Reply:
```suggestion
- docker pull strauman/travis-latexbuild:small # select small texlive scheme
```

Reply:
But docker run will automatically pull the docker image if it's not available locally, right? At least that's what happens for me. So the docker pull doesn't add anything.